### DISCUSSION TITLE
Bring `eslint-plugin-rxjs` into the monorepo?

### DISCUSSION THREAD

**Discussion by reduckted** [ReactiveX/rxjs#7492] [Ideas / Feature request]
The [`eslint-plugin-rxjs`](https://github.com/cartant/eslint-plugin-rxjs) repository hasn't been updated in about 15 months, but still contains some really valuable lint rules. It would be a shame to see it become abandoned. 

Would the rxjs team ever consider bringing it into this rxjs monorepo where it can receive some attention?

**Comment by pumano** [comment]
any news on this?

**Reply by JasonWeinzierl** [reply]
Related discussion in the angular-eslint repo: https://github.com/angular-eslint/angular-eslint/discussions/2094

**Comment by JasonWeinzierl** [comment]
I've released `eslint-plugin-rxjs-x` v1.0.0 with ESLint v10 support, 7 new rules, a strict config, and fixes for all outstanding bugs: https://github.com/JasonWeinzierl/eslint-plugin-rxjs-x/releases/tag/v1.0.0

(For those needing ESLint 8 or 9 support, use versions 0.9.x)
